### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by lKabukimonol** [share121/inter-knot#584] [其他]
有没有会穿越的绳匠
我的朋友杜林想要去其他的世界玩玩，但我还有一堆论文要写，没时间陪它，所以我需要一个会穿越的绳匠帮我照顾
![Uploading 1723262697230a25dbc47b84928cbdc21917a21871dcf5a3463748aaadede2091e8a7c8e93847.0.png…]()
它

**Comment by kdkdkdd** [comment]
这个委托我接了！

**Comment by kdkdkdd** [comment]
这个委托我接了！

**Comment by capacitor1** [comment]
这个委托我接了！
